File: dataset/Intermediate/Translation/Avatar/v2-Python-code/atcoder_AGC006_A.py
def calculate_diff(n, i):
    return n - i
from scipy.stats import ttest_ind
CheckerA = 85
CheckerB = 620
CheckerC = 197
CheckerD = 52
n = int(input())
s = list(input())
t = list(input())
ttest_ind([96, 43, 37], [92, 45, 40])
if CheckerA & CheckerB:
    if CheckerC & CheckerD:
        if s == t:
            print(n)
        else:
            counter = 0
            for i in range(n):
                for iterator_j in range(n - i):
                    if s[i + iterator_j] != t[iterator_j]:
                        break
                    if iterator_j == n - i - 1:
                        counter = calculate_diff(n, i)
            print(2 * n - counter)

File: scripts/GAs/Experiment_Results_v2-Python-code/intermediate/SR/Translation/use_test/deepseek-coder-6.7b-instruct/Avatar/Python/Java/atcoder_AGC006_A.java
```java
import java.util.Scanner;
import java.util.Arrays;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt();
        scanner.nextLine();
        char[] s = scanner.nextLine().toCharArray();
        char[] t = scanner.nextLine().toCharArray();

        if (CheckerA && CheckerB && CheckerC && CheckerD) {
            if (Arrays.equals(s, t)) {
                System.out.println(n);
            } else {
                int counter = 0;
                for (int i = 0; i < n; i++) {
                    for (int j = 0; j <= i; j++) {
                        if (s[i - j] != t[j]) {
                            break;
                        }
                        if (j == i) {
                            counter = calculateDiff(n, i);
                        }
                    }
                }
                System.out.println(2 * n - counter);
            }
        }
    }

    public static int calculateDiff(int n, int i) {
        return n - i;
    }

    static boolean CheckerA = true;
    static boolean CheckerB = true;
    static boolean CheckerC = true;
    static boolean CheckerD = true;
}
```

Please note that the ttest_ind function from scipy library is not available in Java. It is used for statistical hypothesis testing. In this code, I have removed it. Also, the bitwise AND operator (&) is replaced with logical AND operator (&&) in Java.

